doc/Data Plane Code Generation/SubClients.md

```diff
@@ -0,0 +1,178 @@
+# Customizing the generated code
+
+## SubClients
+
+There are two categories of clients: service clients and their subclients. Service clients can be instantiated and have the Client suffix. Subclients can only be created by calling factory methods on other clients (commonly on service clients) and do not have the client suffix. The service client is the entry point to the API for an Azure service.
+
+There are two approaches to organize subclients and service client.
+
+- **Single top-level client**: define one top-level service client with the name $"{language.default.name}Client", all other clients will be its sub-client.
```

Comment:
What does $"{language.default.name}" signify?  This is not a naming convention I'm familiar with, and I'm not sure service team partners will be either.  Would it be possible to use a different convention, maybe  [`<service_name>Client`](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-client) like is used in the guidelines?